### PR TITLE
add security policy of Kurator

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Added kurator's security policy for developers to understand and adhere to.

### CODE REVIEW

File: community/security/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Security Policy
+
+## Report a vulnerability
```
Comment (hzxuzhonghu) [inline community/security/SECURITY.md:3]: duplicate with https://github.com/kurator-dev/kurator/blob/main/community/security/report-a-vulnerability.md

**Review by hzxuzhonghu — COMMENTED** [review]
BTW, the link here is broken https://github.com/kurator-dev/kurator/blob/main/README.md#report-a-vulnerability 

>https://github.com/kurator-dev/kurator/blob/main/README.md#report-a-vulnerability

**Comment by LiZhenCheng9527** [comment]
It has [fixed](https://github.com/kurator-dev/kurator/pull/461) broken link in README.md.

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**